app/controllers/api/questions_controller.rb

```diff
@@ -259,7 +260,8 @@ def normalize_type(type)
       'Multiple Choice' => 'Question::Traditional',
       'Select All That Apply' => 'Question::SelectAllThatApply',
       'Scenario' => 'Question::Scenario',
-      'Stimulus Case Study' => 'Question::StimulusCaseStudy'
+      'Stimulus Case Study' => 'Question::StimulusCaseStudy',
+      'Upload' => 'Question::Upload'
```

Comment:
Could we display this type as "File Upload" to match with Canvas' question type options?